tests/discussions.test.js

```diff
@@ -0,0 +1,364 @@
+            done();
+        });
+    });
+
+    it("GET /Discussions Title", (done) =>{
+        chai.request(server).get("/discussions/title/¿Qué opinan")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("array");
+            response.body.length.should.be.eq(2);
+            done();
+        });
+    });
+
+    it("GET /Discussions Title Not Found", (done) =>{
+        chai.request(server).get("/discussions/title/Canciones")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("GET /Discussions Title Bad Request", (done) =>{
+        chai.request(server).get("/discussions/title/    ")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("GET /Discussions Tracing", (done) =>{
+        chai.request(server).get("/discussions/tracing/6168cf9563929f8f000c7614")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("array");
+            response.body.length.should.be.eq(2);
+            done();
+        });
+    });
+
+    it("GET /Discussions Tracing Not Found", (done) =>{
+        chai.request(server).get("/discussions/tracing/6168d4975471a4bcc2b17446")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("GET /Discussions Tracing Bad Request", (done) =>{
+        chai.request(server).get("/discussions/tracing/6168cf9")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+});
+
+describe("GET Tests Discussion",() =>{
+    before((done) =>{
+        const login = {
+            username: "MiroStar",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("GET /Discussions/ID", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("GET /Discussions/ID", (done) => {
```